gcloud/bigquery/test__helpers.py

```diff
@@ -41,7 +41,8 @@ def test_w_millis(self):
         import datetime
         import pytz
         from gcloud.bigquery._helpers import _total_seconds
-        NOW = datetime.datetime.utcnow().replace(tzinfo=pytz.utc)
+        NOW = datetime.datetime(2015, 7, 29, 17, 45, 21, 123456,
+                                tzinfo=pytz.utc)
```

Comment:
Seems like a good choice for tests. Was the `utcnow()` causing issues or you just wanted a deterministic test?